examples/E2E-73/android/app/src/main/java/com/analyticsreactnativee2e73/MainApplication.kt

```diff
@@ -11,14 +12,14 @@ import com.facebook.react.defaults.DefaultReactHost.getDefaultReactHost
 import com.facebook.react.defaults.DefaultReactNativeHost
 import com.facebook.react.flipper.ReactNativeFlipper
 import com.facebook.soloader.SoLoader
+import com.segmentanalyticsreactnative.AnalyticsReactNativePackage
 
 class MainApplication : Application(), ReactApplication {
 
   override val reactNativeHost: ReactNativeHost =
       object : DefaultReactNativeHost(this) {
         override fun getPackages(): List<ReactPackage> {
           // Packages that cannot be autolinked yet can be added manually here, for example:
-          // packages.add(new MyReactNativePackage());
```

Comment:
Did you intentionally remove this? Above line reads oddly now.

Reply:
Should reset this thing. This is from the boilerplate of RN example apps.